CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 /kebab/ @overlisted
 /baklava/ @amitojsingh366
-/dolma/ @HoloPanio
+/dolma/ @HoloPanio @goldyydev
```

Comment:
i don't think u should add yourself to code owners, leave the final dolma authority to @HoloPanio as he made it